src/libs/logos.js

```diff
@@ -18,30 +18,13 @@ import rareXLogoWhite from 'src/images/brands/rareX/rarex-logo-white.svg'
 import terraLogo from 'src/images/brands/terra/logo.svg'
 import terraLogoWhite from 'src/images/brands/terra/logo-grey.svg'
 import terraLogoShadow from 'src/images/brands/terra/logo-wShadow.svg'
+import {
+  getEnabledBrand, isAnvil, isBaseline, isBioDataCatalyst, isDatastage, isElwazi, isFirecloud, isProjectSingular, isRareX, isTerra
+} from 'src/libs/brand-utils'
 import colors from 'src/libs/colors'
-import { isAnvil, isBaseline, isBioDataCatalyst, isDatastage, isElwazi, isFirecloud, isProjectSingular, isRareX, isTerra } from 'src/libs/config'
 import * as Utils from 'src/libs/utils'
-import { nonBreakingHyphen } from 'src/pages/library/common'
 
 
-export const getAppName = ({ longName = false, capitalizeThe = false } = {}) => {
-  const maybeCapitalizedThe = capitalizeThe ? 'The' : 'the'
-
-  return Utils.cond(
-    [isFirecloud(), () => 'FireCloud'],
-    [isDatastage(), () => 'DataStage'],
-    [isAnvil(), () => longName ? `${maybeCapitalizedThe} NHGRI AnVIL (Genomic Data Science Analysis, Visualization, and Informatics Lab-space)` : 'AnVIL'],
-    [isBioDataCatalyst(), () => 'NHLBI BioData Catalyst'],
-    [isBaseline(), () => longName ? `${maybeCapitalizedThe} Baseline Health Study Data Portal` : 'Project Baseline'],
-    [isElwazi(), () => longName ? `${maybeCapitalizedThe} eLwazi Open Data Science Platform` : 'eLwazi'],
-    [isProjectSingular(), () => 'Project Singular'],
-    [isRareX(), () => `${maybeCapitalizedThe} RARE${nonBreakingHyphen}X Data Analysis Platform`],
-    () => longName ? 'Terra Community Workbench' : 'Terra'
-  )
-}
-
-export const returnParam = () => getAppName().toLowerCase()
-
 const pickBrandLogo = (color = false) => Utils.cond(
```

Comment:
It seems like the brand logo should go into the brand structure.

Reply:
My overall thinking on where things should go is as follows:

- If it's a _config_-like thing (stuff currently in `brands.js` that doesn't involve Javascript functions, etc.) that can be changed/added to, which in turn would make it possible for a non-software engineer, we put it in `brands.js`.
- If it's branding-specific but involves knowing some Javascript, it goes into `brand-utils.js`.

I was punting on moving the logo stuff into `brand-utils.js` because I wanted to think more holistically about what else can be made _config_-like as far as images go but I can go ahead and move the two functions `pickBrandLogo` and `brandLogoMaker` now.

Reply:
Changed in [9d11924](https://github.com/DataBiosphere/terra-ui/pull/3257/commits/9d11924c69e01df4057dd2b51b770eb6f3e1c02e).